### DISCUSSION TITLE
Should the nightly rtl_433 with default settings be able to pick up my R900 meter?

### DISCUSSION THREAD

**Discussion by dimatx** [merbanan/rtl_433#2464] [Q&A] [answered]
Hi all,

I have a Neptune R900 V4 (Model R900M) water meter. Historically, I've been able to use [rtlamr](https://github.com/bemasher/rtlamr) project. Looking to consolidate so I can capture everything I need with rtl_433.

Is my meter expected to work using default settings on the current nightly as long as the "[228]  Neptune R900 flow meters" protocol is enabled?

I tried it and didn't see any captures, but perhaps it's user error.

Thanks!

-----------
In case it's helpful, this is how I'm using rtlamr today.

```
'[10:12:09] INFO: Starting rtl_tcp daemon...
Found 1 device(s):
  0:  Realtek, RTL2838UHIDIR, SN: 00000001
Using device 0: Generic RTL2832U OEM
Found Rafael Micro R820T tuner
[R82XX] PLL not locked!
Tuned to 100000000 Hz.
[10:12:09] INFO: Log level set to INFO
[10:12:09] INFO: Waiting for rtl_tcp...
Allocating 15 zero-copy buffers
[10:12:09] INFO: Handing over control to AMR2MQTT...
Signal caught, exiting!
Signal caught, exiting!
INFO:root:Starting rtlamr
INFO:root:Connecting to MQTT broker at mqtt.********.com:1883
Allocating 15 zero-copy buffers
10:12:11.019577 decode.go:45: CenterFreq: 912380000
10:12:11.019692 decode.go:46: SampleRate: 262144
10:12:11.019696 decode.go:47: DataRate: 32768
10:12:11.019731 decode.go:48: ChipLength: 8
10:12:11.019734 decode.go:49: PreambleSymbols: 32
10:12:11.019737 decode.go:50: PreambleLength: 512
10:12:11.019741 decode.go:51: PacketSymbols: 116
10:12:11.019758 decode.go:52: PacketLength: 1856
10:12:11.019762 decode.go:59: Protocols: r900
10:12:11.019765 decode.go:60: Preambles: 00000000000000001110010101100100
10:12:11.019769 main.go:111: GainCount: 29
```

**Comment by zuckschwerdt** [comment, marked as answer]
Yes, Neptune R900 (not sure about "V4") should work, see #2180 for some hints and experience from other users. Maybe contact them by mentioning theirs usernames here.

**Reply by dimatx** [reply]
Thanks Christian. Will give it another shot and will reopen and tag those users if I'm still stuck.

**Reply by dimatx** [reply]
Just wanted to confirm this is working as expected. Thanks again.

**Reply by mattfox27** [reply]
Is this still working for you?  How did you setup the config?

**Reply by jdevelop** [reply]
Not sure if this is ok to hijack someone's else question - but I just pulled the most recent Docker image (`hertzg/rtl_433:master`) and I have no output for my R900 meter. I confirmed that `rtlamr` can pick things up just fine, so I am curious what exactly am I doing wrong here:

```
> docker run --device /dev/bus/usb/001/012 hertzg/rtl_433:master -Y classic -f 912380000 -s 250k -R 228
rtl_433 version 23.11 (2023-11-28) inputs file rtl_tcp RTL-SDR with TLS

New defaults active, use "-Y classic -s 250k" if you need the old defaults

Found Rafael Micro R820T/2 tuner
Exact sample rate is: 250000.000414 Hz
[SDR] Using device 0: Realtek, RTL2838UHIDIR, SN: 00000001, "Generic RTL2832U OEM"
Allocating 15 zero-copy buffers

```

That's it, no other output

**Reply by zuckschwerdt** [reply]
To debug run with  `-v -Y autolevel -M level -M noise`

**Reply by jdevelop** [reply]
Thanks, the noise was quite high so I got another antenna and now everything is okay.

**Reply by dylanmaniatakes** [reply]
> Not sure if this is ok to hijack someone's else question - but I just pulled the most recent Docker image (`hertzg/rtl_433:master`) and I have no output for my R900 meter. I confirmed that `rtlamr` can pick things up just fine, so I am curious what exactly am I doing wrong here:
> 
> ```
> > docker run --device /dev/bus/usb/001/012 hertzg/rtl_433:master -Y classic -f 912380000 -s 250k -R 228
> rtl_433 version 23.11 (2023-11-28) inputs file rtl_tcp RTL-SDR with TLS
> 
> New defaults active, use "-Y classic -s 250k" if you need the old defaults
> 
> Found Rafael Micro R820T/2 tuner
> Exact sample rate is: 250000.000414 Hz
> [SDR] Using device 0: Realtek, RTL2838UHIDIR, SN: 00000001, "Generic RTL2832U OEM"
> Allocating 15 zero-copy buffers
> ```
> 
> That's it, no other output


I actually have had good success just using sudo rtl_433 -f 915M -H 200 -C si -F "mqtt:homeassistantip,user=mqttuser,pass=pass,events=rtl_433[/model][/id]" it reads my r900 meter about every minute. unfortuanltly my meter seems to be set to roll over at 10k gal so the data is worthless short term because we don't use a ton of water
